README.md

```diff
@@ -13,9 +13,6 @@ import it:
 
 `import 'package:tweet_ui/tweet_ui.dart';`
 
-If you want to show tweets with videos: check the
-[video_player installation](https://pub.dev/packages/video_player#installation) site.
```

Comment:
Maybe we could point to: https://jhomlala.github.io/betterplayer/#/install for instructions on better_player?

Reply:
Added